src/apps/content-editor/src/app/views/ItemEdit/PublishState.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useMemo } from "react";
+import { useParams } from "react-router-dom";
+import moment from "moment";
+import { DataGridPro, GridValueGetterParams } from "@mui/x-data-grid-pro";
+import { Box, Button, Chip } from "@mui/material";
+import { theme } from "@zesty-io/material";
+import { ThemeProvider } from "@mui/material/styles";
+import { WithLoader } from "@zesty-io/core/WithLoader";
+import { instanceApi } from "../../../../../../shell/services/instance";
```

Comment:
We use webpack alias for `shell`, `utility` and `apps` to avoid relative paths. https://github.com/zesty-io/manager-ui/blob/master/src/shell/webpack.config.js#L72

i.e. This can be an import path of `shell/services/instance`